bundles/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/UniFiBindingConstants.java

```diff
@@ -95,7 +95,18 @@ public final class UniFiBindingConstants {
     public static final String CHANNEL_PORT_POE_VOLTAGE = "voltage";
     public static final String CHANNEL_PORT_POE_CURRENT = "current";
 
-    // List of access point channels
+    // List of general device channels
+    public static final String CHANNEL_DEV_ONLINE = "online";
+    public static final String CHANNEL_DEV_STATE = "state";
+    public static final String CHANNEL_DEV_NAME = "name";
+    public static final String CHANNEL_DEV_SITE = "site";
+    public static final String CHANNEL_DEV_MAC_ADDRESS = "macAddress";
+    public static final String CHANNEL_DEV_IP_ADDRESS = "ipAddress";
+    public static final String CHANNEL_DEV_UPTIME = "uptime";
+    public static final String CHANNEL_DEV_LASTSEEN = "lastSeen";
+    public static final String CHANNEL_DEV_EXPERIENCE = "experience";
```

Comment:
I'll look into that. Probably we could just reuse the existing ones. I named these DEV because they also might be shared with e.g. Switch things if they are added at some point whereas the enable and upcoming led channels cannot be used for any device but only for access points. I'd we just reuse the existing constants for now we don't need any DEV constants.

Reply:
I created separate types in the thing-type.xml (which is good I guess) so I also made separate constants here (which is nonsense I guess). So I will just use the existing ones instead.

I now made an extra block for the common channels between clients and devices to keep the logic behind the blocks consistent.

Reply:
> I created separate types in the thing-type.xml (which is good I guess)

Actually, the idea behind channel types is to be able to reuse them for different channels, so it would be better and easier to maintain if you would reference the existing channel types when they are for the same purpose. That's also why there are system channel types. Please see other comments, as labels and/or descriptions might need to be overwritten in some cases (when mentioning _device_ rather than _client_).